visualization/attribute_sliders.js

```diff
@@ -0,0 +1,47 @@
+const sliderAttributes = [
```

Comment:
Since we are normalizing all of these values client side, at some point we should think about the finalized list that we present to the user and if there is a max # items to the list